examples/date-after-dynamic/profile.json

```diff
@@ -0,0 +1,50 @@
+{
+  "schemaVersion": "0.1",
+  "fields": [
+    {
+      "name": "earlier"
+    }, {
+      "name": "later"
+    }
```

Comment:
No big deal, but I think it's a little confusing that "earlier" and "later" are used as field names. At first I confused them for part of the syntax. Maybe just call them field1 and field2?